app/src/main/java/org/fossasia/openevent/app/event/attendees/AttendeesPresenter.java

```diff
@@ -37,11 +37,6 @@ public void attach() {
     }
 
     @Override
-    public void detach() {
```

Comment:
Then please add a null check for subscriber as in all the presenters. It is needed for all subscribers as view can be null anyways even if you don't detach. Please do this and test